src/App.jsx

```diff
@@ -25,21 +26,8 @@ export const alex = {
```

Comment:
Make sure the 'olya' object is defined in the file or imported from another file. If it's not defined, add or import the 'olya' object with the necessary properties.